extras/src/main/java/com/google/gson/typeadapters/RuntimeTypeAdapterFactory.java

```diff
@@ -108,14 +108,14 @@
  *   shapeAdapter.registerSubtype(Circle.class, "Circle");
  *   shapeAdapter.registerSubtype(Diamond.class, "Diamond");
  * }</pre>
- * Finally, register the type adapter in your application's GSON builder:
+ * Finally, register the type adapter factory in your application's GSON builder:
  * <pre>   {@code
  *   Gson gson = new GsonBuilder()
- *       .registerTypeAdapter(Shape.class, shapeAdapter)
+ *       .registerTypeAdapterFactory(Shape.class, shapeAdapterFactory)
```

Comment:
@JakeWharton You're right. That line should be `.registerTypeAdapterFactory(shapeAdapterFactory)`.

How would you rather have me summit the fix (since it's already been merged)? A new PR with this change only?